src/display/api.js

```diff
@@ -720,6 +726,7 @@ var PDFPageProxy = (function PDFPageProxyClosure() {
     this.transport = transport;
     this.stats = new StatTimer();
     this.stats.enabled = getDefaultSetting('enableStats');
+    this.canvasFactory = transport.canvasFactory;
```

Comment:
Based on https://github.com/mozilla/pdf.js/pull/8002#discussion_r99022111. this line should be removed.